frontend/packages/helm-plugin/integration-tests/support/step-definitions/helm/helm-navigation.ts

```diff
@@ -6,16 +6,29 @@ import {
   pageTitle,
   switchPerspective,
 } from '@console/dev-console/integration-tests/support/constants';
-import { catalogPO } from '@console/dev-console/integration-tests/support/pageObjects/add-flow-po';
+import {
+  catalogPO,
+  helmChartRepositoriesPO,
+} from '@console/dev-console/integration-tests/support/pageObjects/add-flow-po';
 import {
   navigateTo,
   addPage,
   catalogPage,
   perspective,
+  createForm,
 } from '@console/dev-console/integration-tests/support/pages';
 import { topologyPage } from '@console/topology/integration-tests/support/pages/topology/topology-page';
 import { helmPage, helmDetailsPage } from '../../pages';
 
+const deleteChartRepositoryFromDetailsPage = (name: string, type: string) => {
+  cy.log(`Deleting ${name}`);
+  cy.byLegacyTestID('kebab-button').click();
+  cy.byTestActionID(`Delete ${type}`).click();
+  createForm.clickConfirm();
+  cy.wait(10000);
```

Comment:
Can we avoid this using timeout

Reply:
at times it takes some time for the resource to get deleted